docs/clients/grpc/projections/README.md

```diff
@@ -0,0 +1,665 @@
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+Enabling a projection that does not exists results in an error: 
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#EnableNotFound
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+ enableProjection(projectionName: string, options?: EnableProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## Disable a projection 
+ 
+Disables a projection, this will save the projection checkpoint.
+You must have access to a projection to disable it.
+
+:::warning 
+The .net clients prior to version 21.6 had an incorrect behavior: they _will not_ save the checkpoint  
+:::
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#Disable
+
+</xode-block>
+<xode-block title="Java">
+
+
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+disableProjection(projectionName: string, options?: DisableProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+Disabling a projection that does not exists results in an error:
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#DisableNotFound
+
+</xode-block>
+<xode-block title="Java">
+
+
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+disableProjection(projectionName: string, options?: DisableProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+
+## Delete a projection
+
+Deletes a projection
+
+<xode-group>
+<xode-block title="C#">
+
+```csharp
+// This is currently not available in the .net client
+```
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+deleteProjection(projectionName: string, options?: DeleteProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## Abort a projection
+
+Aborts a projection, this will not save the projection's checkpoint.
+
+:::warning
+The .net clients prior to version 21.6 had an incorrect behavior: they _will_ save the checkpoint. 
```

Comment:
```suggestion
The .Net clients before version 21.6 had an incorrect behaviour: they _were_ saving the checkpoint. 
```

Reply:
P.S. I'm not sure if that's correct, as 21.6 .NET gRPC client is not yet released (https://www.nuget.org/packages/EventStore.Client.Grpc.Streams)